src/index.html

```diff
@@ -22,6 +22,377 @@
 
   <body>
-    <h1>Catalog</h1>
+    <!-- <h1>Catalog</h1> -->
+    <header class="header">
+      <img
+        src="images/logo.png"
+        alt="Moyo logo image"
+        class="header__logo"
+      />
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              class="nav__link is-active"
+              href="#section-1"
+            >
+              APPLE
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#section-2"
+            >
+              SAMSUNG
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#section-3"
+            >
+              SMARTPHONES
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#section-4"
+              data-qa="nav-hover"
+            >
+              LAPTOPS & COMPUTERS
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#section-5"
+            >
+              GADGETS
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#section-6"
+            >
+              TABLETS
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#section-7"
+            >
+              PHOTO
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#section-8"
+            >
+              VIDEO
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div class="container">
+        <div
+          class="card"
+          data-qa="card"
+        >
+          <img
+            src="./images/imac.jpeg"
+            alt="mac-book image"
+            class="card__image"
+          />
+          <div class="card__product-name">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </div>
+          <div class="card__product-code">Product code: 195434</div>
+          <div class="card__review">
+            <div class="card__review-stars card__review-stars--4">
+              <span class="card__review-star"></span>
+              <span class="card__review-star"></span>
+              <span class="card__review-star"></span>
+              <span class="card__review-star"></span>
+              <span class="card__review-star"></span>
+            </div>
+            <span class="card__review-quantity">Reviews: 5</span>
+          </div>
+          <div class="card__price">
+            <span class="card__price-title">Price:</span>
+            <span class="card__price-cost">$2,199</span>
```

Comment:
The price '$2,199' is repeated across multiple cards. If this is intentional, ensure that each card is meant to display the same product with the same price. Otherwise, consider using different prices for each card.